virttest/libvirt_vm.py

```diff
@@ -302,13 +302,21 @@ def undefine(self, options=None):
         """
         Undefine the VM.
         """
-        # If the current machine contains nvram, we'll set --keep-nvram
+        from virttest.utils_libvirt import libvirt_bios
```

Comment:
Usually we don't import modules in the method, just put it at the beginning of this module

Reply:
Thanks for reminding me, I was aware of this rule, it was my oversight. I'll put it in the correct place in the next update.